website/docs/guides/customize-schema-alias.md

```diff
@@ -0,0 +1,486 @@
+description: "Learn how to properly adjust your generate_schema_name() and generate_alias_name() macros."
+displayText: Learn how to adjust your generate schema name and generate alias name.
+hoverSnippet: Learn how to adjust your generate schema name and generate alias name.
+# time_to_complete: '30 minutes' commenting out until we test
+icon: 'guides'
+hide_table_of_contents: true
+level: 'Advanced'
+recently_updated: true
+---
+
+<div style={{maxWidth: '900px'}}>
+
+## Introduction
+This guide explains how to customize the [schema](/docs/build/custom-schemas) and [alias](/docs/build/custom-aliases) naming conventions in dbt to fit your data warehouse governance and design needs.
+When we develop dbt models and run or build them, objects get created in the data warehouse.
+
+In the rest of the article, for consistency, when we refer to database, we'll be referring to a Google Cloud Platform  project or to a Databricks catalog.
+
+The default behavior of what object is created is the following:
+
+- The database where the object is created will be the database configured at the [environment level in dbt Cloud](/docs/dbt-cloud-environments), or in the `profiles.yml` in dbt Core.
+
+- The schema depends on whether a [custom schema](/docs/build/custom-schemas) has been defined for the model:
+    - If no custom schema is defined, the object will be created in the schema configured in dbt Cloud (`dbt_username` for dev and the default schema for deployment environments) or the schema in the `profiles.yml` in dbt Core.
+ 
+
+    - Note that for automated CI jobs, the schema name is derived from the job number and PR number: `dbt_cloud_pr_<job_id>_<pr_id>`
+    - If a custom schema has been defined, it will concatenate the schema from above with the custom one.
+    - For example, if the configured schema is `dbt_myschema` and the custom one is `marketing`, the objects will be created under `dbt_myschema_marketing`.
+
+- The object name depends on whether an [alias](/reference/resource-configs/alias) has been defined on the model:
+    - If no alias is defined, the object will be created with the same name as the model, without the `.sql` or `.py` at the end.
+    - If an alias is defined, the object will be created with the configured alias.
+
+Those default rules are a great starting point, and many people stick with those without customizing them.
+
+It allows developers to work in their own test schemas (sandboxes) and makes it possible to have different feature requests updating the same models without conflicting with one another.
+
+
+## How to customize this behavior
+
+While the default behavior would fit the needs of most dbt users, there are occasions where the approach doesn’t fit with the existing data warehouse governance and design.
+
+For example, dbt expects that it has the rights to create a schema on the fly _quickly_ (and we recommend that the users running dbt have this ability), but it might not be allowed at your company.
```

Comment:
I don't understand the "_quickly_" here

Reply:
```suggestion
For example, dbt expects that it has the rights to create a schema on the fly (and we recommend that the users running dbt have this ability), but it might not be allowed at your company.
```